starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -979,9 +979,13 @@ sub check_bright_perigee {
         }
     }
 
-    # Pass 1 to _guide_count as third arg to use the count_9th mode
-    my $bright_count = sprintf("%.1f",
-        call_python("utils._guide_count", [ \@mags, $self->{ccd_temp}, 1 ]));
+    # Pass 1 to _guide_count as fourth arg to use the count_9th mode
+    my $bright_count = sprintf(
+        "%.1f",
+        call_python(
```

Comment:
`call_python` allows use of kwargs, so both `date` and `count_9th` would be good as kwargs. Otherwise just looking at this code it is not obvious what the `1` is indicating.

Reply:
Shouldn't they all just be kwargs?  I confess I was confused by the args vs kwargs choices with the call_python implementation.

Reply:
Looks good now.